docs/reference/ml/ml-shared.asciidoc

```diff
@@ -458,6 +458,18 @@ that moment in time. See
 This check runs only on real-time {dfeeds}.
 end::delayed-data-check-config[]
 
+tag::delayed-data-check-config-check-frequency[]
+The how often should the {dfeed} search for late data.
+It must be smaller than the delayed data `check_window`.
+Additionally, if {dfeed} frequency is set, `check_frequency` must
+be larger than `frequency`.
+Delayed data checks cannot occur more often than the configured `frequency`.
+But, setting `check_frequency` higher will increase the time between delayed
```

Comment:
If I'm interpreting it correctly, this seems like a "no duh" sentence, so I'd suggest omitting this one too:
> But, setting `check_frequency` higher will increase the time between delayed